dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.42/V2_42_43__Add_oauth2_server_entity_tables.sql

```diff
@@ -0,0 +1,114 @@
+-- OAuth2 Client table
+create table if not exists oauth2_authorization
+(
+    id                            bigint                      not null primary key,
+
+    uid                           character varying(11)       not null unique,
+    code                          character varying(50) unique,
+    created                       timestamp without time zone not null,
+    lastUpdated                   timestamp without time zone not null,
+    lastupdatedby                 bigint                      ,
+    createdby                     bigint                      ,
```

Comment:
Aren't these missing the FK link to the user table (same in other tables below)?